authentication/esignet-integration-impl/src/test/java/io/mosip/authentication/esignet/integration/service/IdaAuditPluginImplTest.java

```diff
@@ -95,11 +95,11 @@ public void logAudit_WithValidStatus_ThenPass() throws Exception {
                 new ParameterizedTypeReference<ResponseWrapper>() {
                 };
         Mockito.when(authTransactionHelper.getAuthToken()).thenReturn("authToken");
-        Mockito.when(objectMapper.writeValueAsString(any())).thenReturn("requestBody");
-        Mockito.when(restTemplate.exchange(
-                Mockito.any(RequestEntity.class),
-                Mockito.eq(responseType)
-        )).thenReturn(responseEntity);
+//        Mockito.when(objectMapper.writeValueAsString(any())).thenReturn("requestBody");
+//        Mockito.when(restTemplate.exchange(
```

Comment:
if it's not required can we not delete the lines instead of commenting @pvsaidurga